converters/toZigbee.js

```diff
@@ -1248,6 +1248,32 @@ const converters = {
     // #endregion
 
     // #region Non-generic converters
+    ias_wd: {
```

Comment:
This can be moved to the generic-converters region.

Reply:
Done